config/management/src/validator_config.rs

```diff
@@ -95,8 +95,8 @@ impl ValidatorConfig {
         let validator_config_script = transaction_callback(
             owner_account,
             consensus_key.to_bytes().to_vec(),
-            raw_enc_validator_address,
-            raw_fullnode_address.into(),
+            validator_addresses,
+            lcs::to_bytes(&vec![raw_fullnode_address]).unwrap(),
```

Comment:
s/unwrap/map_err ?

Reply:
lcs to bytes... this is pretty unfallible